Updates/Instances/540_shattered_halls.sql

```diff
@@ -458,6 +458,7 @@ REPLACE INTO `creature_template_addon` (`entry`, `mount`, `bytes1`, `b2_0_sheath
 INSERT INTO `creature_linking` (`guid`, `master_guid`, `flag`) VALUES
 -- link "leaders" to bosses
 (@CGUID+74, @CGUID+82, 1024), -- Shattered Hand Legionnaire -> Grand Warlock Nethekurse
+(@CGUID+77, @CGUID+82, 1024), -- Shattered Hand Legionnaire -> Grand Warlock Nethekurse
```

Comment:
I think this is correct and in shattered halls there is really a respawn linking that should prevent the groups from respawning, might be worth investigation though as there are many more groups to come